test-suite/output/Sint63Syntax.out

```diff
@@ -0,0 +1,66 @@
+2%sint63
+     : PrimInt63.int
```

Comment:
Is there a way of printing `int` here instead?

Reply:
Maybe using a notation. Not sure it's worth it though.

Reply:
As a user, I would prefer it. It doesn't make sense to me that unsigned ints are printed as `int` and signed ints are printed as `PrimInt.int`, specially because `PrimInt` is used in the definition of both.

Reply:
A solution is to remove the type definition of `int` in `Sint63` (it's the same type than in `Int63`) and Require *Export* Int63 rather than just require Import it.